.goreleaser.yml

```diff
@@ -117,6 +109,7 @@ dockers:
     - "policies"
     - "coprocess"
     - "tyk.conf.example"
+  skip_push: {{ if .Env.SKIP_DOCKER_PUSH == 'true' }}true{{ else }}false{{ end }}
```

Comment:
Why not have `{{ .Env.SKIP_DOCKER_PUSH }}`?

Reply:
yea, we could use that. tried this since env var comes as string